src/libs/actions/Session.js

```diff
@@ -120,7 +121,7 @@ function signIn(password, exitTo, twoFactorAuthCode) {
         useExpensifyLogin: true,
         partnerName: CONFIG.EXPENSIFY.PARTNER_NAME,
         partnerPassword: CONFIG.EXPENSIFY.PARTNER_PASSWORD,
-        partnerUserID: credentials.login,
+        partnerUserID: normalizeLogin(credentials.login),
```

Comment:
Hmm wait yea. `GetAccountStatus` _should_ return `response.normalizedLogin` so we can probably just use that for the credential instead of whatever the user entered and set it here...

https://github.com/Expensify/Expensify.cash/blob/3aa4992f5e5a73db4d161ddc2dd30710103137a8/src/libs/actions/Session.js#L84-L90

Reply:
Confirmed that `API.GetAccountStatus` does return a `normalizedLogin` that would work for this case.

Are you suggesting calling `API.GetAccountStatus` in signin before calling `API.authenticate`? If so, is it worth adding in another API call when we can easily normalize the login without one?

Reply:
Sorry, to clarify I am saying that I believe we _always_ call `GetAccountStatus` before the call the `Authenticate` that uses `credentials.login`. So if we use the `normalizedLogin` when setting the `{login}` value for `ONYXKEYS.CREDENTIALS` it could perhaps fix this issue.

Reply:
Just so that I'm understanding here, it's better to do this in `GetAccountStatus` bc other calls to that endpoint would _each_ need to then call `normalizedLogin` correct? In other words, we want `normalizedLogin` to occur as far upstream as possible to keep things streamlined. Right?

Reply:
Not quite. We don't need to call `normalizeLogin` at all in the front end code.

Let's follow the user journey when attempting to sign in and maybe that will clear things up. 

The sign in flow works like this...

1. User enters their email or phone
2. We call `GetAccountStatus` and save the `credentials.login` (using the user input not response from `GetAccountStatus` like I am suggesting)
3. User enters a password and signs in
4. We use the user input login and not the normalized login to sign in the user

There is only one place where we call `signIn()` and it happens immediately after `GetAccountStatus`.

Reply:
Ah ok, I gotcha. Thanks for clearing that up for me 👍